frontend/public/components/nav/items.tsx

```diff
@@ -87,7 +88,8 @@ export class ResourceNSLink extends NavLink<ResourceNSLinkProps> {
 
   get to() {
     const { resource, activeNamespace } = this.props;
-    return formatNamespacedRouteForResource(resource, activeNamespace);
+    const lastNamespace = localStorage.getItem(LAST_NAMESPACE_NAME_LOCAL_STORAGE_KEY);
+    return formatNamespacedRouteForResource(resource, lastNamespace !== null ? lastNamespace : activeNamespace);
```

Comment:
I'd say the same here: we should only use `lastNamespace` if it's all projects. Otherwise the nav could switch your project context from one project to a different project, which is confusing.